www-client/seamonkey/seamonkey-2.53.22.ebuild

```diff
@@ -0,0 +1,522 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+WANT_AUTOCONF="2.1"
+
+PYTHON_COMPAT=( python3_{10..11} )
```

Comment:
Is there really no support for `>3.11`?

Reply:
Also IIRC python-3.10 support has been dropped in Gentoo, so I'll remove it as merging. Thanks!